### PR TITLE
Move kubefed to the federation section.

### PR DESCRIPTION
**Pull request by madhusudancs**

### CODE REVIEW

File: release-1.5/release-notes-draft.md [hunk 1]
```diff
@@ -5,12 +5,12 @@
 ## Major Themes
 
 - Improved Federation Support
+  - New command: `kubefed`
```
Comment (nikhiljindal) [inline release-1.5/release-notes-draft.md]: I will call out that the objective of the new CLI is to make it easier to deploy federation control plane.

**Comment by idvoretskyi** [comment]
ping @luxas as he suggested the original version.

**Comment by luxas** [comment]
Totally lgtm, I just wanted to point out that kubefed is a new command, which is still preserved here. 

And yeah, it makes sense to have it in the federation section

**Comment by idvoretskyi** [comment]
LGTM.

File: release-1.5/release-notes-draft.md (hunk 1 above)
Comment (madhusudancs) [inline release-1.5/release-notes-draft.md]: @nikhiljindal it is mentioned in the `Features` section below. I see this section more or less like table of contents.

**Pull request merged**